yarn.lock

```diff
@@ -686,53 +686,53 @@
     "@nodelib/fs.scandir" "2.1.5"
     fastq "^1.6.0"
 
-"@nomicfoundation/edr-darwin-arm64@0.3.8":
```

Comment:
Do any of the yarn updates cause new dependencies / dependencies being updated at the contract level. That should not be the case as they've already been audited

Reply:
No, this is just part of the hardhat upgrade. Nothing code related.